app.js

```diff
@@ -21,3 +27,59 @@ submit.addEventListener('click', () =>{
     const valueTwo = userPronounInput.value;
     userPronouns.textContent = valueTwo;
 });
+
+function changeColorRed() {
```

Comment:
Your radio buttons already have a value property that has the color you want, so you could get rid of all the functions completely. Rather than have all the functions, you could do something like this in your submit event listener to apply the color based on that value:
submit.addEventListener('click', () =>{
    if (buttonOne.checked) {
        bgColor.style.backgroundColor = buttonOne.value;
    } else if (buttonTwo.checked) {
        bgColor.style.backgroundColor = buttonTwo.value;
    } else if (buttonThree.checked) {
        bgColor.style.backgroundColor = buttonThree.value;
    } else if (buttonFour.checked) {
        bgColor.style.backgroundColor = buttonFour.value;
    }

Reply:
A quick edit: it wouldn't be bgColor since you defined that in your function you would want:

submit.addEventListener('click', () =>{
    if (buttonOne.checked) {
        tagColorOne.style.backgroundColor = buttonOne.value;
        tagColorTwo.style.backgroundColor = buttonOne.value;
    } else if (buttonTwo.checked) {
        tagColorOne.style.backgroundColor = buttonTwo.value;
        tagColorTwo.style.backgroundColor = buttonTwo.value;
    } else if (buttonThree.checked) {
        tagColorOne.style.backgroundColor = buttonThree.value;
        tagColorTwo.style.backgroundColor = buttonThree.value;
    } else if (buttonFour.checked) {
        tagColorOne.style.backgroundColor = buttonFour.value;
        tagColorTwo.style.backgroundColor = buttonFour.value;
    }